src/plot-recipes.jl

```diff
@@ -16,31 +16,58 @@
 ### Code:
 
 using RecipesBase
+using Makie
```

Comment:
I'm definitely ***NOT*** going to accept a 100x increase in loading time.  `Makie` is a *massive* dependency.  Isn't there a better way to do this?  `Plots.jl` usess `RecipesBase.jl`, which is very lightweight.

Reply:
I tried using MakieCore, but that only supports the PointBased plots. What could be done is to do the general conversion without the errorbar/band conversion explicitly 🤔 

I'll give that a try

Reply:
Can you not try adding Makie as an optional dependency using `[extras]` in the `Project.toml`? You can see example of this done in Molly.jl where GLMakie is used this way (see files src/Molly.jl and ./Project.toml in the repo).

Reply:
Yep, that is the solution I found as well by now and wanted to try. I just haven't found the time yet and, to be honest, forgot about it, so thank you very much.

The `[extras]` uses `Requires.jl` for exactly this purpose and that should totally work for us.

Reply:
Using `Requires.jl` I now added all original functionality of this PR, while loading only should `Makie.jl` be loaded in the session. So we don't have the forever load time of `Makie`.